superset-frontend/src/views/datasetList/DatasetList.tsx

```diff
@@ -218,7 +268,7 @@ class DatasetList extends React.PureComponent<Props, State> {
                     className="action-button"
                     onClick={confirmDelete}
                   >
-                    <i className="fa fa-trash" />
+                    <TrashIcon />
```

Comment:
If we did do something like `<Icon name="trash" />` here, we could also consider employing SVG sprite tactics.